xarray/tests/test_dataarray.py

```diff
@@ -693,7 +693,7 @@ def test_isel_fancy(self):
         da.isel(time=(('points',), [1, 2]), x=(('points',), [2, 2]),
                 y=(('points',), [3, 4]))
         np.testing.assert_allclose(
-            da.isel_points(time=[1], x=[2], y=[4]).values.squeeze(),
+            da.isel(time=[1], x=[2], y=[4]).values.squeeze(),
```

Comment:
This does not give the same result.
I think we can just ignore the warning by `@pytest.mark.filterwarnings`.